### PR TITLE
Cylindrically Rotate 90 degrees

### LINKED ISSUE #5464: Write a TypeScript program to cylindrically rotate a matrix by 90 degrees
### Description

Write a TypeScript program to cylindrically rotate a matrix by 90 degrees

```
Input  :
1 2 3
4 5 6
7 8 9

Output :
7 4 1
8 5 2
9 6 3
```
<details>
<summary>How to contribute</summary>

- Save the solution in `CylindricallyRotateAMatrixBy90Degrees.ts` file
- Add `CylindricallyRotateAMatrixBy90Degrees.ts` file in `program/cylindrically-rotate-a-matrix-by-90-degrees` folder

</details>

### PR DESCRIPTION
**Pull request by anandfresh**
Issue : https://github.com/codinasion/codinasion/issues/4114

### Why:

Closes codinasion/codinasion#4114

### CODE REVIEW

File: program/cylindrically-rotate-a-matrix-by-90-degrees/CylindricallyRotateAMatrixBy90Degrees.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const rotateClockwise = (matrix: number[]): number[] => {
```
Comment (victoriacheng15) [inline program/cylindrically-rotate-a-matrix-by-90-degrees/CylindricallyRotateAMatrixBy90Degrees.ts:1]: you have an incorrect type for the parameter and the output as well.

**Review by victoriacheng15 — CHANGES_REQUESTED** [review]
you have an incorrect type for the parameter and the output as well.

File: program/cylindrically-rotate-a-matrix-by-90-degrees/CylindricallyRotateAMatrixBy90Degrees.ts (hunk 1 above)
Comment (anandfresh) [reply program/cylindrically-rotate-a-matrix-by-90-degrees/CylindricallyRotateAMatrixBy90Degrees.ts:1]: @victoriacheng15  Its a numeric array , so we should use the number[] for the input & output data type .

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**